src/configuration/customers/login-as-customer.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Login as Customer
+---
+
+There are times when customers need help with their order. To see what they see, store administrators can "Login as Customer," which allows them to see what the customer sees and take actions on their behalf.
+
+Any actions taken while logged in "as Customer" will be applied to the actual customer's account.
+
+## To enable Login as Customer:
+
+Login as Customer is enabled in the Access Control List.
+
+1. Go to **System** > Permissions > **User Roles**.
+1. Select a Role to edit.
+1. Select Role Resources > Role Resouces. Change to 'Custom'.
+1. Under 'Customer', there are options for:
```

Comment:
This isn't really a directive, just a statement. 

```suggestion
1. Scroll to the  **Customers**  parent item and the **Login as Customer** item underneath. Then select the resources that you want to enable for the role:
```

I would put in a screen here that shows an example -- a cut out and not the whole tree, just the ones you want them to know about it.